src/main/java/cubicchunks/server/PlayerCubeMap.java

```diff
@@ -523,9 +531,9 @@ public final void setPlayerViewRadius(int newViewDistance) {
 
 			long playerAddress = getAddress(playerCubeX, playerCubeY, playerCubeZ);
 
-			if (newViewDistance > oldViewDistance) {
+			if (newHorizontalViewDistance > oldHorizontalViewDistance) {
```

Comment:
You now need to handle 4 cases instead of 2, unless you generalize them:
```
newHDist>oldHDist && newVDist>oldVDist
newHDist<oldHDist && newVDist>oldVDist
newHDist>oldHDist && newVDist<oldVDist
newHDist<oldHDist && newVDist<oldVDist
```

Probably best to generalize it and let CubeSelector handle that.